libs/wingii/src/util.rs

```diff
@@ -3,15 +3,16 @@ extern crate serde_json;
 
 pub mod package_json {
 	use std::{
+		collections::HashSet,
 		fs,
 		path::{Path, PathBuf},
 	};
 
 	use crate::node_resolve::{is_path_dependency, resolve_from};
 
-	pub fn dependencies_of(package_json: &serde_json::Value) -> Vec<String> {
+	pub fn dependencies_of(package_json: &serde_json::Value) -> HashSet<String> {
```

Comment:
BTW, what happens if two separate JSII modules have the same transitive dependency? Are we loading the transitive once or multiple times?